cp-ksqldb-server/include/etc/confluent/docker/ksql-server.properties.template

```diff
@@ -1,4 +0,0 @@
-{% set kr_props = env_to_props('KSQL_', '') -%}
```

Comment:
instead we are directly relying on https://github.com/confluentinc/ksql/blob/master/config/ksqldb-server.properties.template